build.gradle

```diff
@@ -168,12 +166,6 @@ project(':springboottrades-web') {
 		compile "org.springframework.boot:spring-boot-starter-thymeleaf:${springBootVersion}"
 		compile "org.springframework.boot:spring-boot-starter-security:${springBootVersion}"
 		compile "org.thymeleaf.extras:thymeleaf-extras-springsecurity4:${thymeleafExtras}"
-
-		//runtime
-		runtime "org.webjars:bootstrap:${bootstrapVersion}"
```

Comment:
I specified CDN sources in header_fragment.html.

On Mon, Nov 30, 2015 at 6:11 AM, David Ferreira Pinto <
notifications@github.com> wrote:

> In build.gradle
> <https://github.com/dpinto-pivotal/cf-SpringBootTrader/pull/43#discussion_r46135905>
> :
>
> > @@ -168,12 +166,6 @@ project(':springboottrades-web') {
> >  		compile "org.springframework.boot:spring-boot-starter-thymeleaf:${springBootVersion}"
> >  		compile "org.springframework.boot:spring-boot-starter-security:${springBootVersion}"
> >  		compile "org.thymeleaf.extras:thymeleaf-extras-springsecurity4:${thymeleafExtras}"
> > -
> > -		//runtime
> > -		runtime "org.webjars:bootstrap:${bootstrapVersion}"
>
> Where are the bootstrap and query coming from? Spring Boot? If so do we
> need to control versions?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dpinto-pivotal/cf-SpringBootTrader/pull/43/files#r46135905>
> .
>



-- 
Corby Page
Senior Field Engineer, Pivotal
713.584.3168